package.json

```diff
@@ -389,7 +388,7 @@
   "scripts": {
     "mage": "./bin/mage",
     "build": "yarn build:cljs && yarn build:js",
-    "build-embedding-sdk-package": "yarn && yarn build-release:cljs && yarn embedding-sdk:generate-package && yarn embedding-sdk:generate-nextjs-compat && yarn build-release:embedding-sdk-package && yarn build-embedding-sdk-cli && yarn embedding-sdk:fixup-types-imports",
+    "build-embedding-sdk-package": "yarn embedding-sdk:clean:dist && yarn embedding-sdk:clean:tsbuildinfo && yarn && yarn build-release:cljs && yarn embedding-sdk:generate-package && yarn embedding-sdk:generate-nextjs-compat && yarn build-release:embedding-sdk-package && yarn build-embedding-sdk-cli && yarn embedding-sdk:dts:generate",
```

Comment:
`dts:generate` already calls `yarn embedding-sdk:clean:tsbuildinfo`. is it redundant that we call it twice?